File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/O. Pyramid/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/O

using System;
using static System.Console;

namespace Codeforces.Assuit.Contest1
{
    class ProblemO
    {
        static void Main(string[] args)
        {
            ushort n = ushort.Parse(Console.ReadLine());
            for(int i=1; i <= n; i++){
                for(int j=0; j<i; j++)
                    Write("*");
                WriteLine();
            }
        }
    }
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/O. Pyramid/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/O
#include<iostream>
#include<cstdio>

using namespace std;
 
int main(){
    unsigned short n;
    cin >> n;
    for(int i=1; i <= n; i++){
        for(int j=0; j<i; j++)
            cout << "*";
        cout << endl;
    }
}